index.js

```diff
@@ -47,24 +51,14 @@ exports.init = function (ssb, config) {
     ssb.friends.hopStream({ old: true, live: true }),
     pull.drain((hops) => {
       for (const dest of Object.keys(hops)) {
-        const value = hops[dest]
-        // myself or friendly peers
-        if (value >= 0) {
-          ssb.ebt.request(dest, true)
-          ssb.ebt.block(ssb.id, dest, false)
-        }
-        // blocked peers
-        else if (value === -1) {
-          ssb.ebt.request(dest, false)
-          ssb.ebt.block(ssb.id, dest, true)
-        }
-        // unfollowed/unblocked peers
-        else if (value < -1) {
-          ssb.ebt.request(dest, false)
-        }
+        requestManager.add(dest, hops[dest])
```

Comment:
This now should be much easier to reason about, everything not related to us (blocking) goes directly to ebt blocking and everything related to us goes request manager 👍